docs/sources/tempo/traceql/_index.md

```diff
@@ -204,52 +198,100 @@ Find traces where the average duration of the spans in a trace is greater than `
 avg(duration) > 20ms
 ```
 
-For example, find traces that have more than 3 spans with an attribute `http.status` with a value of `200`:
+For example, find traces that have more than 3 spans with an attribute `http.status_code` with a value of `200`:
 
 ```
-{ span.http.status = 200 } | count() > 3
+{ span.http.status_code = 200 } | count() > 3
 ```
 
 ## Arithmetic
 
 TraceQL supports arbitrary arithmetic in your queries. This can be useful to make queries more human readable:
 ```
-{ span.bytesProcessed > 10 * 1024 * 1024 }
+{ span.http.request_content_length > 10 * 1024 * 1024 }
 ```
 to compare the ratios of two span attributes:
 ```
-{ span.bytesProcessed < span.jobsProcessed * 10 }
+{ span.bytes_processed < span.jobs_processed * 10 }
 ```
 or anything else that comes to mind.
 
 ## Examples
 
-Find any trace with a span attribute or resource attribute `namespace` set to `prod`:
+### Find traces of a specific operation
+
+Let's say that you want to find traces of a specific operation to find a resource with the `frontend` service name and using the name `POST /api/order`. In the example below, both the resource and the name must be present for any results to be returned.  
 
 ```
-{ .namespace = "prod" }
+{resource.service.name = "frontend" && name = "POST /api/orders"}
 ```
 
-Find any trace with a resource attribute `namespace` set to `prod`:
+When using the same Grafana stack for multiple environments (e.g., `production` and `staging`) or having services that share the same name but are differentiated though their namespace, the query looks like:
 
 ```
-{ resource.namespace = "prod" }
+{
+  resource.service.namespace = "ecommerce" &&
+  resource.service.name = "frontend" &&  
+  resource.deployment.environment = "production" && 
+  name = "POST /api/orders"
+}
+```
+
+### Find traces having a particular outcome
+
+This example finds all traces on the operation `POST /api/orders` that have an erroneous root span:
+
+```
+{
+  resource.service.name="frontend" && 
+  name = "POST /api/orders" && 
+  status = error
+}
+```
+
+This example finds all traces on the operation `POST /api/orders` that return with an HTTP 5xx error:
+
 ```
+{
+  resource.service.name="frontend" && 
+  name = "POST /api/orders" && 
+  span.http.status_code >= 500
+}
+```
+
+### Find traces that have a particuliar behavior
+
+You can use query filtering on multiple spans of the traces.
+This example locates all the traces of the `GET /api/products/{id}` operation that access the database. It's a convenient request to identify caching problems.
+
+```
+{span.service.name="frontend" && name = "GET /api/products/{id}"} && {.db.system="postgresql"}
+```
+
+### Find traces going through `production` and `staging` instances
+
+This example finds traces that go through `production` and `staging` instances. It convenient to identify misconfigurations and leaks across environments. 
```

Comment:
```suggestion
This example finds traces that go through `production` and `staging` instances. 
It's a convenient request to identify misconfigurations and leaks across production and non-production environments. 
```

@knylander-grafana please review

Reply:
Much better. Thank you for adding the clarification (and catching the missing "is").